### PR TITLE
Add backblaze svc

### PR DESCRIPTION
**Pull request by paragm**
Add Backblaze backup service to the block list

this change allows to access the main backblaze website and the help site but blocks the actual service

### CODE REVIEW

File: services/backblaze.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+id: backblaze
+name: Blackblaze
+rules:
+  - '@@||www.backblaze.com^'
```
Comment (Sergey-Lyapin) [inline services/backblaze.yml:4]: not sure about these exceptions

File: services/backblaze.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
....454-1.141 0-.875-.828-1.098-1.555-1.098h-1.57v2.239zm-2.31-4.192h3.91c1.833 0 3.833.387 3.833 2.664 0 1.168-.727 1.969-1.805 2.313v.027c1.375.18 2.297 1.2 2.297 2.563 0 2.187-2.074 2.918-3.942 2.918H62.23V8.617zm9.54 0h2.312v8.352h4.277v2.133h-6.59z" /><path d="M82.59 11.668l-1.274 3.258h2.563l-1.29-3.258zm-.887-3.05h1.91l4.559 10.484H85.57l-.906-2.223H80.59l-.871 2.223h-2.547l4.531-10.485zm6.832 8.351l5.317-6.309h-5.125V8.617h7.921v2.043l-5.21 6.309h5.39v2.133h-8.293zm9.82-8.352h7.125v2.133h-4.816v1.953h4.547v2.133h-4.547v2.133h5.082v2.133h-7.39z" /><path d="M4.555-.027s3.422 2.992 2.957 5.644c-.207 1.18-1.774 3.13-.93 4.98.914 1.997 1.387 5.844-2.723 4.848C-.195 14.465.121 9.5 3.133 6.88 6.187 4.223 5.156 1.3 4.555-.027" /><path d="M7.68 8.945c-.227 2.02 2.007 4.032.234 6.766-1.23 1.894.656 4.555 3.078 4.07 2.656-.531 3.313-2.21 3.242-4.16-.093-2.476-1.078-3.164-3.27-5.023-2.37-2.012-.538-5.809-.538-5.809S7.934 6.723 7.68 8.945" /><path d="M.465 14.855c.75.922 2.2 2.008 4.976 2.07.84.016.184 1.583 1.864 2.993 1.453 1.223 3.41 1.25 4.457.855.61-.226 1.183-.445.742.122-1.973 2.527-5.399 2.367-7.844 1.453-2.906-1.086-3.637-3.223-4.113-4.555-.512-1.445-.25-3.145-.082-2.938" /></svg>
```
Comment (Sergey-Lyapin) [inline services/backblaze.yml:8]: please use this svg instead:
```
<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="-4.5 0 23 23"><path d="M4.55-.03s3.43 3 2.96 5.65c-.2 1.18-1.77 3.13-.93 4.98.92 2 1.39 5.84-2.72 4.85C-.2 14.46.12 9.5 3.13 6.88 6.2 4.22 5.16 1.3 4.55-.02"/><path d="M7.68 8.95c-.23 2.02 2 4.03.23 6.76-1.23 1.9.66 4.56 3.08 4.07 2.66-.53 3.31-2.2 3.24-4.16-.09-2.48-1.07-3.16-3.27-5.02-2.37-2.01-.53-5.81-.53-5.81s-2.5 1.93-2.75 4.16"/><path d="M.47 14.86c.75.92 2.2 2 4.97 2.07.84.01.18 1.58 1.86 2.99a4.77 4.77 0 0 0 4.46.85c.61-.22 1.19-.44.74.13-1.97 2.52-5.4 2.36-7.84 1.45-2.9-1.09-3.64-3.23-4.11-4.56-.51-1.44-.25-3.14-.08-2.94"/></svg>
```

File: services/backblaze.yml (hunk 1 above)
Comment (paragm) [reply services/backblaze.yml:4]: @Sergey-Lyapin this allows users to visit the main website

**Comment by Sergey-Lyapin** [comment]
Let's continue there:
https://github.com/AdguardTeam/HostlistsRegistry/pull/466

**Pull request closed without merging**